frontend/src/components/Main/Actions/ActionModal.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from "react";
+import {inject, observer} from "mobx-react";
+import {Modal, Row, Col} from "react-bootstrap";
+import SelectInput from "../../common/SelectInput";
+import {optionType} from "../../../constants/mainConstants";
+import PropTypes from "prop-types";
+
+@inject("mainStore")
+@observer
+class ActionModal extends Component {
+    render() {
+        const {mainStore} = this.props;
+        return (
+            <Modal
+                show={mainStore.showModal}
+                onHide={() => mainStore.closeActionModal()}
+                size="m"
+                aria-labelledby="contained-modal-title-vcenter"
+                centered>
+                <Modal.Header>
+                    <Modal.Title id="contained-modal-title-vcenter">Download Report</Modal.Title>
+                    <button
+                        id="close-modal"
+                        className="btn btn-danger"
+                        style={{float: "right"}}
```

Comment:
use `float-right` class in bootstrap https://getbootstrap.com/docs/4.0/utilities/float/

Reply:
done